neo/Prompt/Commands/SC.py

```diff
@@ -233,12 +233,11 @@ def execute(self, arguments):
             print("Please open a wallet")
             return False
 
-        if len(arguments) < 6:
+        if len(arguments) < 5:

```

Comment:
Is there a specific reason why you completely removed the ability to add fees? I think that feature is still viable to prioritise deployment over other transactions in the mempool.

Reply:
I will add it back in. I wasn't thinking about the case where a fee wasn't required but was desired for prioritization.